src/App.tsx

```diff
@@ -15,48 +16,117 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  Alphabetically = 'isAlphabeticaly',
+  Length = 'isLength',
+}
+
+type Props = {};
+
+export const App: React.FC<Props> = () => {
+  const [isAlphabeticaly, setIsAlphabeticaly] = useState(false);
+  const [isLength, setIsLength] = useState(false);
+  const [isReverse, setReverse] = useState(false);
```

Comment:
You don't need these states. Actually, you can use just the `sortType` and `isReversed`.